src/snowflake/snowpark/_internal/ast_utils.py

```diff
@@ -859,3 +879,102 @@ def build_udf(
         t = ast.kwargs.add()
         t._1 = k
         build_expr_from_python_val(t._2, v)
+
+
+def build_udtf(
+    ast: proto.Udtf,
+    handler: Union[Callable, Tuple[str, str]],
+    output_schema: Union[
+        StructType, Iterable[str], "PandasDataFrameType"  # noqa: F821
+    ],  # noqa: F821
+    input_types: Optional[List[DataType]],
+    name: Optional[str],
+    stage_location: Optional[str] = None,
+    imports: Optional[List[Union[str, Tuple[str, str]]]] = None,
+    packages: Optional[List[Union[str, ModuleType]]] = None,
+    replace: bool = False,
+    if_not_exists: bool = False,
+    parallel: int = 4,
+    max_batch_size: Optional[int] = None,
+    strict: bool = False,
+    secure: bool = False,
+    external_access_integrations: Optional[List[str]] = None,
+    secrets: Optional[Dict[str, str]] = None,
+    immutable: bool = False,
+    comment: Optional[str] = None,
+    *,
```

Comment:
Aside: To me, one of the mysteries of Python is how positional wildcard parameters fit together with the rest of the parameter list. For instance, why is this here, and not on line 892, where optional parameters start?

Another major mystery of Python is how to evolve function signatures in a backward compatible way.

Reply:
I modeled this after the API signature of `udtf`, but actually it would be much better to simply remove `*`. Don't really see a need for that here. Wydt?

Reply:
I'm fine with removing `*`, because other than ignoring arguments (possibly incorrectly), it doesn't add any functionality.

Reply:
Is ignoring arguments desirable? Do callers of this function forward `*args` and `**kwargs` that they don't understand, and treat as opaque?

Reply:
This is actually a recently touched (Python3.9) feature, some context here https://peps.python.org/pep-0570/#specification. Basically `*` and `/` can be used to overcome some issues with mixing positional and named arguments.